src/rules/prefer-importing-jest-globals.ts

```diff
@@ -0,0 +1,189 @@
+            ) {
+              return fixer.insertTextAfter(
+                firstNode,
+                `\n${createFixerImports(isModule, jestFunctionsToImport)}`,
+              );
+            }
+
+            const importNode = sourceCode.ast.body.find(
+              node =>
+                node.type === AST_NODE_TYPES.ImportDeclaration &&
+                node.source.value === '@jest/globals',
+            );
+
+            if (
+              importNode &&
+              importNode.type === AST_NODE_TYPES.ImportDeclaration
+            ) {
+              const existingImports = importNode.specifiers.reduce<string[]>(
+                (imports, specifier) => {
+                  /* istanbul ignore else */
+                  if (
+                    specifier.type === AST_NODE_TYPES.ImportSpecifier &&
+                    specifier.imported?.name
+                  ) {
+                    imports.push(specifier.imported.name);
+                  }
+
+                  return imports;
+                },
+                [] as string[],
+              );
+
+              const allImports = [
+                ...new Set([...existingImports, ...jestFunctionsToImport]),
+              ];
+
+              return fixer.replaceText(
+                importNode,
+                createFixerImports(isModule, allImports),
+              );
+            }
+
+            const requireNode = sourceCode.ast.body.find(
+              node =>
+                node.type === AST_NODE_TYPES.VariableDeclaration &&
+                node.declarations.some(
+                  declaration =>
+                    declaration.init &&
+                    (declaration.init as any).callee &&
+                    (declaration.init as any).callee.name === 'require' &&
+                    (declaration.init as any).arguments?.[0]?.type ===
+                      'Literal' &&
+                    (declaration.init as any).arguments?.[0]?.value ===
+                      '@jest/globals',
+                ),
+            );
+
+            if (
+              requireNode &&
+              requireNode.type === AST_NODE_TYPES.VariableDeclaration
+            ) {
+              const existingImports =
+                requireNode.declarations[0]?.id.type ===
+                AST_NODE_TYPES.ObjectPattern
+                  ? requireNode.declarations[0]?.id.properties.map(property => {
+                      if (
+                        property.type === AST_NODE_TYPES.Property &&
+                        property.key.type === AST_NODE_TYPES.Identifier
+                      ) {
+                        return property.key.name;
+                      }
+                      /* istanbul ignore else */
+                      if (
+                        property.type === AST_NODE_TYPES.Property &&
+                        property.key.type === AST_NODE_TYPES.Literal
+                      ) {
+                        return property.key.value;
+                      }
```

Comment:
```suggestion
                      if (
                        property.type === AST_NODE_TYPES.Property &&
                        isSupportedAccessor(property.key)
                      ) {
                        return getAccessorValue(property.key);
                      }
```

Reply:
Also you can cover this with [`[() => {}]`](https://github.com/jest-community/eslint-plugin-jest/blob/main/src/rules/utils/__tests__/parseJestFnCall.test.ts#L609-L615)

Reply:
Never seen this syntax but thanks, it covers the program 🙏